5.5/test/run

```diff
@@ -108,15 +110,15 @@ function run_change_password_test() {
   create_container "testpass1" -e MYSQL_USER=user -e MYSQL_PASSWORD=foo \
     -e MYSQL_DATABASE=db -v ${tmpdir}:/var/lib/mysql/data:Z
   test_connection testpass1 user foo
-  docker stop $(get_cid testpass1)
+  docker stop $(get_cid testpass1) >/dev/null
 
   # Create second container with changed password
   create_container "testpass2" -e MYSQL_USER=user -e MYSQL_PASSWORD=bar \
     -e MYSQL_DATABASE=db -v ${tmpdir}:/var/lib/mysql/data:Z
   test_connection testpass2 user bar
 
   # The old password should not work anymore
-  if mysql_cmd "$(get_container_ip testpass2)" user foo -e 'SELECT 1;'; then
+  if mysql_cmd "$(get_container_ip testpass2)" user foo -e 'SELECT 1;' 2>/dev/null; then
```

Comment:
if we fail to connect for whatever reason, the reason will be hidden and it will be pain to debug it ;-)